contrib/flo-scio_2.12/src/main/scala/com/spotify/flo/contrib/scio/ScioOperator.scala

```diff
@@ -67,7 +67,7 @@ class ScioOperator[T] extends TaskOperator[ScioJobSpec.Provider[T], ScioJobSpec[
       spec.pipeline(sc)
 
       // Start job
-      val scioResult = Try(sc.close())
+      val scioResult = Try(sc.run())
```

Comment:
Does this mean we will stop supporting old versions of Scio?

Reply:
So `.close()` is not yet removed from Scio https://github.com/spotify/scio/blob/v0.8.1/scio-core/src/main/scala/com/spotify/scio/ScioContext.scala#L678

Reply:
It is not compatible at byte code level due to method signature change.